frontend/sidedish/src/view/sidedish/card/Card.jsx

```diff
@@ -0,0 +1,66 @@
+import {
+    ImgWrapper,
+    Img,
+    TextContainer,
+    Title,
+    Description,
+    PriceContainer,
+    Price,
+    EventBadge,
+} from "./Card.style";
+
+function EventBadges(eventBadges) {
+    return eventBadges.map((eventBadge, idx) => (
+        <EventBadge
+            key={idx}
+            eventBadgeColor={eventBadge === "런칭특가" ? "orange" : "green"}
+        >
+            {eventBadge}
+        </EventBadge>
+    ));
+}
+
+function Card({
+    title,
+    image,
+    description,
+    fixedPrice,
+    discountPrice,
+    eventBadges,
+}) {
+    const originPrice = (
+        <Price isClientPrice={false}>{fixedPrice.toLocaleString()}원</Price>
+    );
+    const clientPrice = (
+        <Price isClientPrice>
+            {fixedPrice === discountPrice
+                ? fixedPrice.toLocaleString()
+                : discountPrice.toLocaleString()}
+            원
+        </Price>
+    );
+    const eventTags = eventBadges ? EventBadges(eventBadges) : null;
+
+    return (
+        <li>
+            <ImgWrapper>
+                <Img alt={title} src={image} />
+            </ImgWrapper>
+            <div>
+                <TextContainer>
+                    <Title>{title}</Title>
+                    <Description>{description}</Description>
+                    <PriceContainer>
+                        <>
+                            {clientPrice}
+                            {fixedPrice !== discountPrice && originPrice}
```

Comment:
```suggestion
                            {isDiscounted && originPrice}
```

의도를 잘 드러내는 변수로 분리하면 한 눈에 의미를 파악하기가 더 쉬워질 거 같아요.